build/Build.cmd

```diff
@@ -81,13 +81,15 @@ IF "%APPVEYOR_REPO_TAG%" == "true" (goto :sign)
 copy /y %temp%\pom.xml.original pom.xml
 
 if %ERRORLEVEL% NEQ 0 (
-    @echo Build SparkCLR Scala components failed, stop building.
-    popd
-    goto :eof
+  @echo Build SparkCLR Scala components failed, stop building.
+  popd
+  goto :eof
 )
 
 @echo SparkCLR Scala binaries
 copy /y target\*.jar "%SPARKCLR_HOME%\lib\"
+@REM Exclude any original pre-shaded .jar files
```

Comment:
yes - we only want the shaded version under "runtime". How about changing - 
copy /y target\*.jar "%SPARKCLR_HOME%\lib\" 
To - 
copy /y target\spark*.jar "%SPARKCLR_HOME%\lib\"